packages/sanity/src/_internal/cli/actions/app/undeployAction.ts

```diff
@@ -19,33 +20,43 @@ export default async function undeployCoreAppAction(
   // Check that the project has a Core application ID
   let spinner = output.spinner('Checking application info').start()
 
+  const appId =
+    cliConfig && '__experimental_coreAppConfiguration' in cliConfig
+      ? cliConfig.__experimental_coreAppConfiguration?.appId
+      : undefined
+
+  if (!appId) {
+    spinner.fail()
+    output.print(`No Core application ID provided.`)
+    output.print(
+      'Please set `__experimental_coreAppConfiguration` in sanity.cli.js or sanity.cli.ts.',
```

Comment:
```suggestion
      'Please set appId `__experimental_coreAppConfiguration` in sanity.cli.js or sanity.cli.ts.',
```

Should this specify that the appId in particular is missing?